src/libDirectoryService/DirectoryService.h

```diff
@@ -291,6 +291,9 @@ class DirectoryService : public Executable, public Broadcastable
 
     uint8_t CalculateNewDifficulty(const uint8_t& prevDifficulty);
 
+    /// Calculate and return the min size of required each shard for a specifc number of total nodes
+    uint32_t CalculateShardSize(const uint32_t numberOfNodes);
```

Comment:
We can make this static in case our test code needs to use this without actually going through a DS instance